react/features/chat/components/web/Chat.js

```diff
@@ -73,6 +74,9 @@ class Chat extends AbstractChat<Props> {
             this._scrollMessageContainerToBottom(true);
         } else if (this.props._isOpen && !prevProps._isOpen) {
             this._scrollMessageContainerToBottom(false);
+            VideoLayout.onResize();
```

Comment:
Don't we have an action for opening? Maybe we can hook up the resize thing there?

Reply:
Moved the resizing logic in `chat/actions`